# Custom message type represent a reading from a distance sensor used for reading and writing

# The frame_id (name) of this LED
string frame_id
# The current value of this LED
# Values in [0, 1]
float32 value
